sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CAutoConfiguration.java

```diff
@@ -27,6 +28,7 @@
  * and import {@link AADB2COAuth2ClientConfiguration} class for AAD B2C OAuth2 client support.
  */
 @Configuration
+@ConditionalOnResource(resources = "classpath:aadb2c.enable.config")
```

Comment:
Add a test case to check the invoke times of CommonCondition and UserFlowCondition when the resource is on/absent from the class path.